.env.sample

```
@@ -252,3 +255,15 @@ RESEND_API_KEY=re_xxxxxxxxxx
 # DB_ALERT_EMAILS=
 # Boolean: to enable/disable db alerts. This needs the 'utility' microservice
 # DB_ALERT_ENABLE=
+
+
+PROJECT_ID=
+CLIENT_EMAIL=
+PRIVATE_KEY=
+
+CONSUMER_CONFIG_ACK_WAIT=10_000  # IN nanos(10_000)
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)